sklearn/compose/_column_transformer.py

```diff
@@ -49,6 +49,9 @@ class ColumnTransformer(TransformerMixin, _BaseComposition):
 
     .. versionadded:: 0.20
 
+    .. versionchanged:: 0.21
+        This class is no longer an experimental feature.
+
```

Comment:
also not sure if we need this, since it doesn't really show a change, rather a change in status.

Reply:
Agreed, we can remove